src/components/sbb-file-selector/sbb-file-selector.scss

```
@@ -0,0 +1,87 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-file-selector-background-color: var(--sbb-color-white-default);
+  --sbb-file-selector-border-color: var(--sbb-color-cloud-default);
+
+  width: sbb.px-to-rem-build(320);
```

I cannot understand why this works, but I feel like there is a hashtag needed :):
```suggestion
  width: #{sbb.px-to-rem-build(320)};
```
But I'm wondering if this is really wanted in default mode? @mcilurzo

---

Fixed, but still open topic for Manuel

---

Discussed with @kyubisation -> we leave the fixed width for all variants.